tests/singleRequestLink.ts

```diff
@@ -0,0 +1,9 @@
+import { assert } from 'chai';
+import SingleRequestLink from '../src/singleRequestLink';
+
+
+describe('SingleRequestLink', () => {
```

Comment:
I don't think this is needed since the link won't be shipped. If it is, this needs a lot more tests